test/react_client_test.dart

```diff
@@ -34,7 +34,6 @@ main() {
           'children': testChildren,
         }),
       );
-      expect(unconvertJsProps(instance)['style'], new isInstanceOf<Map<String, dynamic>>());
```

Comment:
why was this removed?

Reply:
Good catch - I didn't mean to. I was combining the branch that had the updated test cases with the master branch and this got cut out.